static/flow/views/program-editor-view.js

```diff
@@ -275,16 +275,15 @@ var ProgramEditorView = function(options) {
     //
     base.loadProgramFromSpec = function(params) {
         // console.log("[DEBUG] ProgramEditorView loadProgramFromSpec()", params);
-        var filename    = params.filename;
-        var displayedName    = params.displayedName;
-
         programloaded = true;
         
         var programSpec = params.programdata;
+        var filename    = programSpec.name;
```

Comment:
Looks like this value isn't used at all?

Reply:
Might not be.  Let's save it for the next cleanup round.